test/example.md

```diff
@@ -63,69 +65,58 @@ _Result:_
 }
 ```
 
-
 #### ExampleClass()
 
-
 ExampleClass description
 
-
 #### ExampleClass.method2(num)
 
 - `num`: [`<number>`]
 
 method2 description
 
-
```

Comment:
Two empty lines were supposed to separate methods though I don't have a strong opinion on this one.